README.md

```diff
@@ -23,23 +23,28 @@ The workflow will be set to fail if:
+| `auto-create-project`      | Whether or not SRM should automatically create a specified project name if it does not exist                                                                                                      | `false`       | No<sup>[2]</sup>               |
+| `base-branch-name`         | The parent branch name of a project created in SRM                                                                                                                                                | `undefined`   | No<sup>[2]</sup><sup>[3]</sup> |
+| `target-branch-name`       | The target branch name of a project created in SRM. <br/>SRM automatically creates the branch if it does not exist yet in the project, and the new branch will be created from `base-branch-name` | `undefined`   | No<sup>[2]</sup>               |
+| `source-and-binaries-glob` | A comma-separated-list of file globs matching source and binary files to be packaged and sent to SRM                                                                                              | `undefined`   | No                             |
+| `tool-outputs-glob`        | A comma-separated list of file globs matching tool output/scan result files                                                                                                                       | `undefined`   | No                             |
+| `wait-for-completion`      | Whether to wait for the analysis to complete before exiting                                                                                                                                       | `false`       | No                             |
+| `ca-cert`                  | A custom CA cert to use for HTTPS connections to SRM                                                                                                                                              | `undefined`   | No                             |
+| `dry-run`                  | Whether to submit an analysis (false/undefined) or only test the connection and credentials (true)                                                                                                | `undefined`   | No                             |
 
 **Notes**
 1. Either `project-id` or `project-name` is required. An error will be thrown if neither is specified or both are specified.
-2. `base-branch-name` is required if `target-branch-name` is specified and doesn't exist yet in the project.
+2. If `auto-create-project` is set to true, SRM will automatically create a project that does not exist.
+   The `base-branch-name` will be used as the default branch, and `target-branch-name` will be used as the analysis branch.
+   If neither is specified, it will simply use the default branch.
+   If only one of the branches is specified, it will throw error during project creation.
```

Comment:
"create a project that does not exist" would be better as "create the project if it does not _yet_ exist"